pages/blog/whats-new-in-graphql-cli-4.1.0.mdx

```diff
@@ -0,0 +1,74 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: "What's new in GraphQL CLI 4.1",
+  author: 'ephelan',
+  tags: [
+    'graphql',
+    'node',
+    'codegen',
+    'graphql-cli',
+    'graphql-config',
+    'graphback',
+  ],
+  date: '2020-10-12',
+  description:
+    'GraphQL CLI - experience the modern way to develop full-stack GraphQL applications.',
+  image:
+    'https://user-images.githubusercontent.com/20847995/67651234-85bf1500-f916-11e9-90e5-cb3bd0e6a338.png',
+};
+
+export default withArticle({ ...meta });
+
+[GraphQL CLI](http://graphql-cli.com) is your one-stop shop for developing full-stack GraphQL applications in Node.js. Since the release of 4.0.0, we have added a number of enhancements which will further elevate your GraphQL development experience.
+GraphQL CLI aggregates multiple community projects giving developers the best getting started experience. Tools included in the CLI are mature and developed over the years based on Guild's experience in pushing production ready GraphQL solutions and also through collaboration with Red Hat community projects.
+
+## GraphQL Code Generator
+
+[Add what is new in GraphQL Codegen]
+
+To use GraphQL Code Generator from the CLI, run `graphql codegen` from your application root:
```

Comment:
Maybe add those things later at the end?
Let's focus on the new init and serve and GraphBack stuff, as those are the main things that improved?

About codegen and inspector you can mention later in the blog post that the are included as recommend, best practices workflows and got updated to the latest versions

Reply:
Cool, sounds good.